rs3-full-boss-guides/barrows.txt

```diff
@@ -36,8 +38,8 @@ Generally, Barrows is only camped by high level players in an effort to obtain t
     "color": 39423,
     "fields": [
       {
-        "name": "__Preset Suggestions & Breakdowns__",
-        "value": "⬥ [Preset Sample](https://i.imgur.com/7spEdR7.png)\n⬥ [Breakdown](https://i.imgur.com/iI1h9y1.png)",
+        "name": "__Preset Suggestion__",
+        "value": "⬥ [Preset Sample](https://i.imgur.com/P6kwtXF.png)",
```

Comment:
Update to shareable preset-maker link. I can do this later if it's too big of a hassle